core/amber/src/main/python/core/runnables/main_loop.py

```diff
@@ -149,8 +150,8 @@ def process_control_payload(
 
     def process_input_tuple(self) -> None:
         """
-        Process the current input tuple with the current input link. Send all result
-        Tuples to downstream workers.
+        Process the current input tuple or state with the current input link.
```

Comment:
the method name is `process_input_tuple`, while your comment says it could process state. Please avoid such misaligned comments. You might want to update the method name.

Reply:
It only processes tuples, updated.